dependency-check-suppressions.xml

```diff
@@ -175,4 +175,25 @@
     <gav regex="true">^org\.codehaus\.groovy:groovy-json:.*$</gav>
     <cpe>cpe:/a:apache:groovy</cpe>
   </suppress>
+  <suppress>
+    <notes><![CDATA[
+   file name: tomcat-embed-websocket-8.5.27.jar
+   ]]></notes>
+    <gav regex="true">^org\.apache\.tomcat\.embed:tomcat-embed-websocket:.*$</gav>
+    <cve>CVE-2018-1304</cve>
```

Comment:
You should be able to just add the cve (CVE-2018-1304 and CVE-2018-1305) to the existing suppression around line 106. Up to you